protocol/x/clob/keeper/grpc_stream_orderbook.go

```diff
@@ -10,6 +10,7 @@ func (k Keeper) StreamOrderbookUpdates(
 ) error {
 	err := k.GetFullNodeStreamingManager().Subscribe(
```

Comment:
Can we make a ticket to sanitize + validity check the requests coming in for orderbook streaming?

Also need a ticket to add this to websocket

Reply:
yeah we have tickets https://linear.app/dydx/issue/CT-1103/subaccount-fns-websocket-support and https://linear.app/dydx/issue/CT-968/positions-test-correctness